cloud/azure/keyvault/detectors-keyvault.tf

```diff
@@ -0,0 +1,79 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Add `threshold` function & use `lasting` function instead of aggregation function for Key Vault detectors.